### PR TITLE
feat(rln-relay): pass in index to keystore credentials

### LINKED ISSUE #1774: chore(rln-relay): allow passing in an index to the keystore
### Background
Currently, all credentials are loaded into the keystore path provided, but by default, nwaku chooses the 0th credential in the array - https://github.com/waku-org/nwaku/blob/625787463388e0f8c883ef3749b0e2dfecfbcdcc/waku/v2/waku_rln_relay/group_manager/on_chain/group_manager.nim#L443

### Details
- Use the existing `--rln-relay-membership-index` flag to choose the appropriate credential

### Acceptance criteria
- [ ] Operators can decide which rln credentials they wish to use

### PR DESCRIPTION
**Pull request by rymnc**
# Description

Allows users/operators to pass in the index of credentials to use. "Credentials" represents the keystore provided by the `waku_keystore` module. 
 
# Changes

- [x] `s/rlnRelayMembershipIndex/rlnRelayCredIndex/g` to be more descriptive
- [x] Adds a new config option (`--rln-relay-membership-group-index`) to chat2 and wakunode2 to allow operators to 
select the credential within a an array of 
credentials in rln app

## Issue

closes #1774

### CODE REVIEW

File: tests/v2/waku_rln_relay/test_wakunode_rln_relay.nim [hunk 1]
```diff
@@ -51,7 +51,7 @@ procSuite "WakuNode - RLN relay":
     await node1.mountRlnRelay(WakuRlnConfig(rlnRelayDynamic: false,
       rlnRelayPubsubTopic: rlnRelayPubSubTopic,
       rlnRelayContentTopic: contentTopic,
-      rlnRelayMembershipIndex: some(MembershipIndex(1)),
+      rlnRelayCredIndex: MembershipIndex(1),
```
Comment (vpavlin) [inline tests/v2/waku_rln_relay/test_wakunode_rln_relay.nim:54]: Would it make sense to also rename the type `MembershipIndex`?

**Review by vpavlin — APPROVED** [review]
Small comment, otherwise LGTM

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks!
May kindly elaborate a bit more the PR description to understand better what does a index of credentials represent?

**Review by alrevuelta — APPROVED** [review]

File: tests/v2/waku_rln_relay/test_wakunode_rln_relay.nim (hunk 1 above)
Comment (rymnc) [reply tests/v2/waku_rln_relay/test_wakunode_rln_relay.nim]: This looks like it fell through the sieve, it should actually be <index>.uint instead. fixed in https://github.com/waku-org/nwaku/pull/1777/commits/956a63c8e20bd326626b1e16e2d938c696e28c1d

**Pull request merged**